lib/auth_io.dart

```diff
@@ -374,7 +374,7 @@ Future<AccessCredentials> obtainAccessCredentialsViaCodeExchange(
 class _ServiceAccountClient extends AutoRefreshDelegatingClient {
   final JwtFlow flow;
   AccessCredentials credentials;
-  Client authClient;
+  late Client authClient;
```

Comment:
Can we move the initializing expression?

Reply:
Moved to initializer.